purchase_work_acceptance_invoice_plan/models/work_acceptance.py

```diff
@@ -0,0 +1,21 @@
+
+# Copyright 2019 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class WorkAcceptance(models.Model):
+    _inherit = "work.acceptance"
+
+    wa_in_id = fields.Many2one(
+        comodel_name="purchase.invoice.plan",
+        string="Invoice Plan",
+        readonly=1,
+        copy=False,
+    )
+
+    def button_cancel(self):
```

Comment:
I don't think it is good to remove installment_id here,
even there this WA is cancelled, it should keep the info, that once it was created for this installment_id